doc/js_tutorials/js_setup/js_setup/js_setup.markdown

```diff
@@ -103,3 +103,76 @@ Building OpenCV.js from Source
 
     @note
     It requires `node` installed in your development environment.
+
+-#  [optional] To build `opencv.js` with threads optimization, append `--threads` option.
+
+    For example:
+    @code{.bash}
+    python ./platforms/js/build_js.py build_js --build_wasm --threads
+    @endcode
+
+    The default threads number is the logic core number of your device. You can use `cv.parallel_pthreads_set_threads_num(number)` to set threads number by yourself and use `cv.parallel_pthreads_get_threads_num()` to get the current threads number.
+
+    @note
+    You should build wasm version of `opencv.js` if you want to enable this optimization. 
+
+-#  [optional] To build `opencv.js` with wasm simd optimization, append `--simd` option.
+    To build wasm intrinsics tests, append `--build_wasm_intrin_test` option.
```

Comment:
Move this sentence after following example as it is only useful for development flow.